templates/Procfile

```diff
@@ -1,2 +1,3 @@
+release: bin/auto_migrate
```

Comment:
The `configure_automatic_deployment` method in app_builder.rb adds also adds a `release` process. My impression is that we don't need both.

Reply:
Yup. Wow, this has drifted so much since I started, thank you so much for carefully reviewing these changes!!!

Reply:
I kept the version in app builder so folks could opt-in/out with generators as needed.